### DISCUSSION TITLE
Please use timestamps in backing database rather than chars

### DISCUSSION THREAD

**Discussion by turekg** [requarks/wiki#3971] [General]
I am trying to extract data from the database in order to get a list of pages that have been updated within a given period of time (I believe this is a feature that has been requested in the past). What makes it really a pain is that all dates in the backing DB are going in as chars and in a non standard DB format, so I cannot even use the DB provided functions to convert to timestamps on which to perform search.
Thank you

**Comment by NGPixel** [comment]
The DB is designed to be efficient with JS in general, not for the 1 or 2 people that will query it directly. So timestamps will stay in ISO8601 format, which is standard and stays consistent across languages / platforms.
